extensions/interactions/MultipleChoiceInput/directives/oppia-interactive-multiple-choice-input.component.ts

```diff
@@ -38,6 +38,7 @@ export class InteractiveMultipleChoiceInputComponent implements OnInit {
   @Input() showChoicesInShuffledOrderWithValue: string;
   choices;
   answer;
+  questionIsOnceAnswered: boolean = false;
```

Comment:
```suggestion
  questionIsAnsweredOnce = false;
```
When you initialize a variable with a value, you don't need to mention the type since it gets inferred.

Reply:
Done!